This is broken in the current release and would require a lot of patches to be
backported. Remove this patch for the next release.

--- a/base/memory/tagging.h
+++ b/base/memory/tagging.h
@@ -14,10 +14,6 @@
 #include "base/compiler_specific.h"
 #include "build/build_config.h"
 
-#if defined(ARCH_CPU_ARM64) && defined(__clang__) && \
-    (BUILDFLAG(IS_LINUX) || BUILDFLAG(IS_ANDROID))
-#define HAS_MEMORY_TAGGING 1
-#endif
 
 constexpr int kMemTagGranuleSize = 16u;
 #if defined(HAS_MEMORY_TAGGING)
